tests/validation/test_phenopacket_store_auditor.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from ppktstore.model import PhenopacketStore
+from ppktstore.validation import PhenopacketStoreAuditor, default_auditor
+
+
+class TestPhenopacketAuditor:
+
+    @pytest.fixture(scope="class")
+    def auditor(self) -> PhenopacketStoreAuditor:
+        return default_auditor()
+
+    def test_phenopacket_store_cohort_fail(
```

Comment:
It seems like these tests check `PhenopacketStoreAuditor`, but in reality, they just test `UniquePhenopacketId`.

I am worried that the tests won't scale well, if we extend `PhenopacketStoreAuditor` with another checks. Maybe we should make `UniquePhenopacketId` public and test it separately?

FWIW, it seems like `PhenopacketStoreAuditor` is more like a runner, than an auditor. It makes sense to split testing the Q/C int
1. test the "runner" functionality
2. test the individual checks, such as `UniquePhenopacketId`

For the sake of time, I don't think you need to drop everything and implement `1.`. However, we should be ready for doing that.

So, please make `UniquePhenopacketId` public and test it directly, with the smallest test suite possible.